mephisto/abstractions/architects/router/node/server.js

```diff
@@ -500,6 +501,28 @@ app.post("/log_error", function (req, res) {
   res.json({ status: "Error log sent!" });
 });
 
+app.get("/task_config.json", function (req, res) {
+  const { mephisto_task_version } = req.query;
+  if (mephisto_task_version !== CURR_MEPHISTO_TASK_VERSION) {
+    mephisto_message_queue.push({
+      packet_type: PACKET_TYPE_ERROR,
+      subject_id: SYSTEM_SOCKET_ID,
+      data: {
+        error_type: "version-mismatch",
+        text:
+          "Package `mephisto-task` version mismatch. Expected " +
+          "version " +
+          CURR_MEPHISTO_TASK_VERSION +
+          " but frontend is currently using " +
+          mephisto_task_version +
+          ". This may cause unexpected errors, be sure to " +
+          "update your `mephisto-task` JS package.",
```

Comment:
Perhaps we can give explicit guidance around next steps in the error message:

"...be sure to update your `mephisto-task` dependency with `npm install mephisto-task@latest --save`. If this warning still persists please open an issue at `https://github.com/facebookresearch/Mephisto/issues`"

The latter issue portion is added to cover the case where we somehow miss deploying a package to npm, but the version exists on the `main` branch.